.github/workflows/keep-workflows-enabled.yaml

```diff
@@ -46,6 +46,8 @@ jobs:
           - { repo: seasonal-cov,    workflow: ingest-to-phylogenetic.yaml }
           - { repo: status,          workflow: ci.yaml }
           - { repo: WNV,             workflow: ingest-to-phylogenetic.yaml }
+          - { repo: yellow-fever,    workflow: ci.yaml }
```

Comment:
The [yellow-fever CI](https://github.com/nextstrain/yellow-fever/blob/04b92ca46b3f24b11d47178c088e888dc4b4650b/.github/workflows/ci.yaml#L3-L8) does not have a `schedule` event so I don't think it needs to be included here.

Reply:
Ah, yeah, okay, removed the ci workflow.